### ISSUE TITLE
downace.com [Pop-up]

### ISSUE THREAD

**Issue by xDarkARG** [uBlockOrigin/uAssets#1170]
URL(s) where the issue occurs

`https://downace.com/2My37`

Describe the issue

ADS

Versions

Browser/version: Chrome 62.0.3202.62
uBlock Origin version: 1.14.20

Settings

I use the configuration that comes already in ublock when one installs the Extension

**Comment by mapx-** [comment]
test
`downace.com##a[href*="inclk.com/"]` or `##a[href*="inclk.com/"]`

Which is the real download button ?

**Comment by okiehsch** [comment]
Hm, I don't see `##a[href*="inclk.com/"]`, only `##a[href^="https://9xmovie.co/download/"]`.

**Comment by mapx-** [comment]
it's the overlay on page before adding your rules (at least in FF)

**Comment by mapx-** [comment]
weird, in chrome it' isn't

**Comment by okiehsch** [comment]
OK, I did not test with Firefox, is it still necessary?

**Comment by mapx-** [comment]
well, I still get that clicking overlay even using your filters,m could you test  in FF ?

**Comment by mapx-** [comment]
overlay link: `http://p238993.inclk.com/itab/sa?cid=ADBI_MAIN_IF1DOWNACE&pid=238993&q=FMK1920-MW.rar%20-%20DownAce&ap=fp%3Dundefined%26referrer%3Dhttps%3A%2F%2Fdownace.com%2F2My37%26cmp%3DADBLOCK_POP%26popeye%3DbXg9dW5kZWZpbmVkJm15PXVuZGVmaW5lZCZjeD11bmRlZmluZWQmY3k9dW5kZWZpbmVkJnc9MTU0NCZoPTg2MSZjPTEmcz0xJnQ9MjImaT0wJm89LTYw%26olive%3D1%26cbr%3D1515096000`

**Comment by okiehsch** [comment]
Yes, I can reproduce, weird difference between the browsers, the code is the same,
`downace.com##script:inject(window.open-defuser.js)` only fixes the popup with Chrome.
`downace.com##script:inject(abort-on-property-write.js, u0EE)` works with Chrome and Firefox.

**Comment by mapx-** [comment]
I got `##a[href*="inclk.com/"]` on lots of sites at least in FF

**Comment by okiehsch** [comment]
Maybe EasyList could add it as a generic filter.

**Comment by mapx-** [comment]
@ryanbr

**Comment by okiehsch** [comment]
Or something like `##a[href^="http://inclk.com/itab/"]` if they think `##a[href*="inclk.com/"]` is to broad.

**Comment by ghajini** [comment]
`downace.com/2Pyq3`

`||downace.com/sw.js$first-party`
